src/components/interactive-builder/interactive-builder.component.tsx

```diff
@@ -269,6 +281,19 @@ const InteractiveBuilder: React.FC<InteractiveBuilderProps> = ({ isLoading, onSc
     setShowDeleteQuestionModal(true);
   };
 
+  const getAnswerErrors = (answers: Array<Record<string, string>>) => {
+    const answerLabels = answers?.map((a) => a.label) || [];
+    const errors: Array<ValidationError> = validationResponse.filter((e) => answerLabels?.includes(e.field.label));
+    return errors || [];
+  };
+
+  const getValidationError = (question: Question) => {
+    const errorField: ValidationError = validationResponse.find(
+      (e) => e.field.label === question.label && e.field.id === question.id && e.field.type === question.type,
+    );
+    return errorField?.errorMessage || errorField?.warningMessage || '';
+  };
```

Comment:
I think for now we can just do the error message and leave out the warning message, these might clog the UI and we may not need to represent them if they are not deal-breakers for form-submission. What do you think?

Reply:
@arodidev I have actually not seen a form with `warningMessages` but just to ensure that if they are returned they don't break the UI, I am just returning them if they exist.